rocket_sqlite_oracle/README.md

```diff
@@ -0,0 +1,9 @@
+# Run
+
+Do:
+
+```bash
+$ ROCKET_DATABASES='{oracle_signatures={url="./db/signatures.sqlite"}}' cargo run -- --secret="./dev_keys/oracle_secret"
```

Comment:
Do we still need to pass ` ROCKET_DATABASES`?

I think with moving all of this to a dedicated crate and having a `Rocket.toml` file, that should be unnecessary.

Reply:
No, you need to use the `ROCKET_DATABASES` or add it to your environment variables.